test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -149,24 +149,26 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 		return contactFormBlock.insertSubject( subject );
 	}
 
-	toggleMoreToolsAndOptions() {
-		return driverHelper.clickWhenClickable(
+	async toggleMoreToolsAndOptions() {
+		await driverHelper.clickWhenClickable(
 			this.driver,
-			By.xpath( "//button[@aria-label='More tools & options']" )
+			By.xpath( "//button[@aria-label='More tools & options' or @aria-label='Options']" )
```

Comment:
This was needed due to recenti(ish) GB changes.

Reply:
Since we're on 9 as stable, should we just be using the updated version?
```suggestion
			By.xpath( "//button[@aria-label='Options']" )
```

Reply:
Possibly, I will test it out 👍🏻